.github/workflows/pr.yaml

```diff
@@ -0,0 +1,282 @@
+  #     - name: Cache composer dependencies
+  #       uses: actions/cache@v2
+  #       with:
+  #         path: ${{ steps.composer-cache.outputs.dir }}
+  #         key: ${{ matrix.php }}-composer-${{ hashFiles('**/composer.lock') }}
+  #         restore-keys: ${{ matrix.php }}-composer-
+  #     - name: Validate composer files
+  #       run: composer validate composer.json
+  #     - name: Composer install with exported .env variables
+  #       run: |
+  #         set -a && source .env && set +a
+  #         APP_ENV=dev composer install
+  #     - name: Yarn install
+  #       uses: actions/setup-node@v2
+  #       with:
+  #         node-version: '14'
+  #     - run: yarn install
+  #     - name: Yarn build
+  #       run: yarn run encore production
+  #     - name: Run local server
+  #       run: symfony server:start --no-tls --port=8000 -d
+  #     - name: Run Doctrine Migrations
+  #       run: php bin/console doctrine:migrations:migrate --no-interaction
+  #     - name: Start LibreOffice API container
+  #       run: docker build --tag libreoffice-api ./.docker/libreoffice-api && docker run -d --publish 9980 --name libreoffice-api libreoffice-api
+  #     - name: Update environment to use libreoffice-api container
+  #       # `docker port` writes two lines on Ubuntu, e.g.
+  #       # 0.0.0.0:49160
+  #       # :::49160
+  #       run: echo LIBREOFFICE_API_BASE_URL="http://$(docker port libreoffice-api 9980 | head -1)" >> .env.local && cat .env.local
+  #     - name: Load Fixtures
+  #       run: php bin/console hautelook:fixtures:load --purge-with-truncate --no-interaction
+  #     - name: Cypress
+  #       uses: cypress-io/github-action@v2
+  #       env:
+  #         CYPRESS_baseUrl: http://127.0.0.1:8000
+
+  php-cs-fixer:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '8.1' ]
+    name: PHP Coding Standards Fixer (${{ matrix.php }})
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php}}
+          extensions: apcu, ctype, iconv, imagick, json, redis, soap, xmlreader, zip
+          coverage: none
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ matrix.php }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ matrix.php }}-composer-
+      - name: Install Dependencies
+        run: composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist
+      - name: php-cs-fixer
+        run: phpdbg -qrr ./vendor/bin/php-cs-fixer fix --dry-run
```

Comment:
I find it easier to have the actions run the same checks that you (should) run locally before committing code, e.g.

```sh
docker-compose exec phpfpm composer install
docker-compose exec phpfpm composer coding-standards-check
```

How to run coding standards checks must be documented in `README.md`.